### DISCUSSION TITLE
New sequence discovered in db migration test

### DISCUSSION THREAD

**Discussion by andre-qumulo** [sqlalchemy/alembic#1474] [Usage Questions] [answered]
We are upgrading from SqlAlchemy 1.3 to 2.x, Alembic from 1.4.2 to 1.13.1.

We have a test case that spins up a new DB, runs all of the migrations on it, and then checks to see if the current models we've defined would generate new schemas.

I am getting the following failure:
```
AssertionError: 'No changes in schema detected.' not found in "2024-05-17T21:06:26.152668+0000 : MainThread : alembic.ddl.postgresql : Detected sequence named 'some_table_id_seq' as owned by integer column 'some_table(id)', assuming SERIAL and omitting"
```
The only thing I can tell is that we moved from something like this:
```
BaseModel: DefaultMeta = db.Model

class SomeTableModel(BaseModel):
    __tablename__ = 'some_table'
    id = Column(Integer, primary_key=True)
```
to
```
class SomeTableModel(db.Model):  # type: ignore
    __tablename__ = 'some_table'
    id: Mapped[int] = mapped_column(primary_key=True)
 ```
What's odd is that when I try to generate a new migration file via flask_migrate, I'm not getting any changes so I don't know where that detected sequence is coming from.  

Is this a case where I just need to update my test case?  Is there a way to set something to ignore these so that it says `No changes in schema detected.` ?

**Comment by CaselIT** [comment]
Hi,

That's expected, you are updating a lot of versions at once

I think you may use the include_names function to control what alembic seems, but maybe you are better off using the check command of alembic that will fail of there are changes detected

**Reply by andre-qumulo** [reply]
I'll look into those, thanks!

**Reply by andre-qumulo** [reply, marked as answer]
After some further debugging I think the problem is with `flask-migrate`. It should be passing some values around but don't seem to be.  I'm following up with them here: https://github.com/miguelgrinberg/Flask-Migrate/discussions/545

I appreciate you taking the time to provide some suggestions!
